### ISSUE TITLE
error with geom_line_interactive with filtered dataset

### ISSUE THREAD

**Issue by JMLuther** [davidgohel/ggiraph#43]
Thanks for this great package, I find it to be one of the most useful.
I've run into an issue when creating an interactive Shiny app. When the dataset is filtered down to one group with a single observation, I get the following error: 

> Error in f(...) : could not find function "message_wrap" .

I've recreated a simple example to run just within R:

```
# minimal df
fake <- data.frame(
  id=c(rep(1,5), 2),
  x=c(1:5, 5),
  y=c(rnorm(5, 100,1), 75),
  tip=c(rep("this is the tip", 6)))
)

# works if both groups are included
testPlot <- fake %>% group_by(id) %>% 
  ggplot(aes(x,y,group=id, tooltip=tip, data_id=id)) +
  geom_line_interactive() +
  geom_point_interactive()
ggiraph(code=print(testPlot), hover_css="stroke:orange;stroke-width:3", 
        width = 0.6, selected_css = "stroke:orange;stroke-width:3")

# error if filtered down to single id with only 1 observation
testPlot1 <- fake %>% group_by(id) %>% filter(id==2) %>% 
  ggplot(aes(x,y,group=id, tooltip=tip, data_id=id)) +
  geom_line_interactive() +
  geom_point_interactive()
ggiraph(code=print(testPlot1), hover_css="stroke:orange;stroke-width:3", 
        width = 0.6, selected_css = "stroke:orange;stroke-width:3")

# basic ggplot gives a warning, but still plots the point
basicPlot <- fake %>% group_by(id) %>% filter(id==2) %>% 
  ggplot(aes(x,y,group=id)) +
  geom_line() +
  geom_point()
basicPlot

```
thanks!

**Comment by davidgohel** [comment]
Thanks, it was a bug

**Comment by JMLuther** [comment]
Thanks for the fix. Works great, both in R and Shiny app.

Fyi, when installing update, it was giving me difficulty by trying to install Rcpp from source. To get it to install successfully I had to install without dependencies: `devtools::install_github('davidgohel/ggiraph', dep=FALSE)`
I had installed the dev version previously without this issue, so I'm not sure why now. If others have the same issue I can send more info, but it's working fine for me now.
